Cargo.toml

```diff
@@ -64,4 +64,4 @@ threshold_crypto = { opt-level = 3 }
 
 [patch.crates-io]
 secp256k1-zkp = { git = "https://github.com/dpc/rust-secp256k1-zkp/", branch = "sanket-pr" }
-
+ring = { git = "https://github.com/dpc/ring", rev = "5493e7e76d0d8fb1d3cbb0be9c4944700741b802" }
```

Comment:
Any reason this is tagged to a commit and not a branch? And is this a long term patch?
CC: @dpc

Reply:
We needed some unreleased ring wasm fixes. Maybe we don't need it anymore.

Reply:
Nope. 0.17 still cooking https://crates.io/crates/ring/versions